src/twisted/mail/scripts/mailmail.py

```diff
@@ -8,20 +8,22 @@
 
 from __future__ import print_function
 
+import email.utils
 import os
 import sys
-import rfc822
 import getpass
-from ConfigParser import ConfigParser
-
 try:
-    import cStringIO as StringIO
-except:
-    import StringIO
+    # Python 3
+    from configparser import ConfigParser
+except ImportError:
+    # Python 2
+    from ConfigParser import ConfigParser
```

Comment:
Well I think using six would be a good thing, but the original Twisted Python 3 porting effort avoided the use of third party python3 porting libraries.  In hindsight I think that was a mistaken strategy, but what is done is done.

Reply:
well, we already depend on six...as I guess that all our dependencies depend on six :)

I think that is not to late to start using six, rather than going further on this wrong way :)